hvac/v1/__init__.py

```diff
@@ -385,6 +385,69 @@ def auth_userpass(self, username, password, mount_point='userpass', use_token=Tr
 
         return self.auth('/v1/auth/{}/login/{}'.format(mount_point, username), json=params, use_token=use_token)
 
+    def create_userpass(self, username, password, policies, mount_point='userpass'):
+        """
+        POST /auth/<mount point>/users/<username>
+        """
+
+        # Users can have more than 1 policy. It is easier for the user to pass in the
+        # policies as a list so if they do, we need to convert to a , delimited string.
+        if type(policies) == list:
```

Comment:
Ah cool, wasn't aware of the `*policies` type syntax. Only downside of that is that I'd want it to be a required param where if it was after the `mount_point='userpass'` it would be considered optional, right?